R/verb-expand.R

```diff
@@ -64,6 +54,23 @@ expand.tbl_lazy <- function(data, ..., .name_repair = "check_unique") {
   purrr::reduce(distinct_tables, left_join, by = group_vars(data))
 }
 
+extract_dot_vars <- function(dot) {
```

Comment:
Is the name still applicable, now that this is a top-level function?

Reply:
👍  Renamed to `extract_expand_dot_vars()`.